### ISSUE TITLE
FireDM is not accessible to screen readers on Windows

### ISSUE THREAD

**Issue by jkinnunen** [firedm/FireDM#383]
Currently, FireDM isn't accessible to any screen readers on Windows. I'm using the free and open source screen reader called NVDA. It can be downloaded at https://www.nvaccess.org/download/.

Would it be possible to make FireDM accessible so that visually impaired people like me could use it? That'd be really great.
Thanks.

**Comment by firedm** [comment]
@jkinnunen 
no problem, I will try my best to make it accessible to screen readers

**Comment by jkinnunen** [comment]
Thank you! That would be awesome.

**Comment by Aboghazala** [comment]
@jkinnunen 
unfortunately, FireDM GUI is built by tkinter, which has no support to screen readers

more info in this link: https://sourceforge.net/p/tcl/mailman/tcl-core/thread/8c01c04a-7c44-89a6-e2b4-90e919e37f63%40codebykevin.com/#msg36955189

sorry about this, 
closing issue
